addons/website/static/src/xml/website.seo.xml

```diff
@@ -86,54 +24,108 @@
+            <td class="text-center"><i t-if="widget.used_h2" class="fa fa-check" t-attf-title="{{ widget.keyword }} is used in page second level heading"/></td>
+            <td class="text-center"><i class="js_seo_keyword_title fa fa-check" style="visibility: hidden;" t-attf-title="{{ widget.keyword }} is used in page title"/></td>
+            <td class="text-center"><i class="js_seo_keyword_description fa fa-check" style="visibility: hidden;" t-attf-title="{{ widget.keyword }} is used in page description"/></td>
+            <td class="text-center"><i t-if="widget.used_content" class="fa fa-check" t-attf-title="{{ widget.keyword }} is used in page content"/></td>
+            <td class="js_seo_keyword_suggestion"/>
+            <td class="text-center"><a href="#" class="oe_remove" data-action="remove-keyword" t-attf-title="Remove {{ widget.keyword }}"><i class="fa fa-trash"/></a></td>
         </tr>
     </t>
 
     <t t-name="website.seo_suggestion">
-        <li class="list-inline-item oe_seo_suggestion">
-            <span t-attf-title="#{ widget.tooltip() }" t-attf-class="oe_seo_keyword #{ widget.highlight() } js_seo_suggestion" t-att-data-keyword="widget.keyword">
-                <t t-raw="widget.keyword"/>
-            </span>
+        <li class="list-inline-item">
+            <span class="js_seo_suggestion badge badge-info" t-att-data-keyword="widget.keyword" t-esc="widget.keyword"/>
         </li>
     </t>
 
     <t t-name="website.seo_preview">
-        <li class="oe_seo_preview_g">
+        <div class="oe_seo_preview_g">
             <div class="rc">
                 <div class="r"><t t-esc="widget.title"/></div>
                 <div class="s">
                     <div class="kv"><t t-esc="widget.url"/></div>
                     <div class="st"><t t-esc="widget.description"/></div>
                 </div>
             </div>
-        </li>
+        </div>
+    </t>
+
+    <div t-name="website.seo_meta_title_description">
+        <h4><small>Optimize your referencing in search engines</small></h4>
+        <div class="row">
+            <div class="col-lg-6">
+                <div class="form-group">
+                    <label for="website_meta_title">
+                        Title <i class="fa fa-question-circle-o" title="The title will take a default value unless you specify one."/>
+                    </label>
+                    <input type="text" name="website_meta_title" id="website_meta_title" class="form-control" maxlength="70" size="70"/>
+                </div>
+                <div class="form-group">
+                    <label for="website_meta_description">
+                        Description <i class="fa fa-question-circle-o" title="The description will be generated by search engines based on page content unless you specify one."/>
+                    </label>
+                    <textarea name="website_meta_description" id="website_meta_description" class="form-control" style="height:60px;"/>
```

Comment:
At least 120px

Reply:
We wanted to reduce vertical scroll as much as possible but I guess 60px is fine to gain on usability. This textarea can be resized by the user as needed though.